helm/gomodel/templates/gateway.yaml

```
@@ -0,0 +1,28 @@
+{{- if .Values.gateway.enabled -}}
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: {{ include "gomodel.fullname" . }}
+  labels:
+    {{- include "gomodel.labels" . | nindent 4 }}
+spec:
+  parentRefs:
+    - name: {{ .Values.gateway.parentRef.name }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)